src/manifests/svc.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# Source: https://github.com/ory/k8s/blob/master/helm/charts/kratos/templates/service-admin.yaml
+# helm charm v0.25.3
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ app_name }}-admin
+  labels:
+    app.kubernetes.io/component: {{ app_name }}-admin
+    app.kubernetes.io/name: {{ app_name }}
+    app.kubernetes.io/managed-by: juju
+    app.juju.is/created-by: {{ app_name }}
+    operator.juju.is/name: {{ app_name }}
+    operator.juju.is/target: application
+spec:
+  type: ClusterIP
+  ports:
+    - port: 80
+      targetPort: http-admin
```

Comment:
You should be able to use `4434` instead of `http-admin` as `targetPort` in order to avoid having to patch the `StatefulSet`

Reply:
I am not sure how it could avoid patching the StatefulSet. 4434 and 4433 are not exposed by default. I would still need to patch the statefulset when I use numbers instead of port name in the SVCs

Reply:
I think it will work (I have tried this). According to the docs (https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#ports):
```
Not specifying a port here DOES NOT prevent that port from being exposed. Any port which is listening on the default "0.0.0.0" address inside a container will be accessible from the network
```